### PR TITLE
Update elemental_difference.py

### PR DESCRIPTION
**Pull request by PipKat**
I've added comments to this first file for you to see the types of questions I have in regard to your files. Answers to these questions will allow me to edit other files with more confidence. Thanks!

### CODE REVIEW

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 1]
```diff
@@ -7,43 +7,58 @@
...
-      >>> op.inputs.mesh.connect(my_mesh)
-      >>> my_through_layers = bool()
-      >>> op.inputs.through_layers.connect(my_through_layers)
-
-      >>> # Instantiate operator and connect inputs in one line
-      >>> op = dpf.operators.averaging.elemental_difference(field=my_field,mesh_scoping=my_mesh_scoping,mesh=my_mesh,through_layers=my_through_layers)
-
-      >>> # Get output data
-      >>> result_fields_container = op.outputs.fields_container()"""
+    """Transform an ``ElementalNodal`` or ``Nodal`` field into an ``Elemental`` field.
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:13]: We need a one-line description for display in the table. A longer description can follow after a blank line.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 2]
```diff
@@ -7,43 +7,58 @@
...
-      >>> op = dpf.operators.averaging.elemental_difference(field=my_field,mesh_scoping=my_mesh_scoping,mesh=my_mesh,through_layers=my_through_layers)
-
-      >>> # Get output data
-      >>> result_fields_container = op.outputs.fields_container()"""
+    """Transform an ``ElementalNodal`` or ``Nodal`` field into an ``Elemental`` field. 
+    
+    Each elemental value is the maximum difference between the computed result for all 
+    nodes in this element. The result is computed on a given element scoping.
+
+    Parameters
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:18]: We use a "Parameters" section to list the inputs. I'm not sure how you were formatting these. They appear to be names (so strings). For collapse_shell_layers, you indicate it's Boolean but have a default of ``None``. Is this equivalent to ``True`` or ``False``? You didn't have the last two parameters listed (config and server), so I added them. They both have defaults of ''None``, in which case what happens?

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 3]
```diff
@@ -7,43 +7,58 @@
...
+    through_layers : bool, optional
+    
+    config : optional
+        The default is ``None``.
+    server : optional
+        The default is ``None``.
+        
+    Returns
+    -------
+    :class:`ansys.dpf.core.fields_container`
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:35]: I was told that classes do not return anything--yet you had an "available outputs" section. Consequently, I formatted as a Returns section--using the class directive that I was taught and then a general description for the return type.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 4]
```diff
@@ -59,12 +74,12 @@ def __init__(self, field=None, mesh_scoping=None, mesh=None, through_layers=None
 
     @staticmethod
     def _spec():
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:76]: I realize an underscore preceding the method means that it is not visible. I just wanted the doc strings to match what was in the visible doc.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 5]
```diff
@@ -76,7 +91,7 @@ def default_config():
 
     @property
     def inputs(self):
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:93]: Alex says that a property is described as a noun, so I've made these changes for your review and approval.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 6]
```diff
@@ -99,22 +114,23 @@ def outputs(self):
 #internal name: elemental_difference
 #scripting name: elemental_difference
 class InputsElementalDifference(_Inputs):
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:116]: I've never seen an input parameter be linked like this. I assume it is OK and that no "Parameters" section with  description is needed here.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 7]
```diff
@@ -128,9 +144,9 @@ def __init__(self, op: Operator):
 
     @property
     def field(self):
-        """Allows to connect field input to the operator
+        """Field inputs to connect to the operator.
 
-        - pindoc: field or fields container with only one field is expected
+        - pindoc: A field or fields container with only one field is expected.
```
Comment (PipKat) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:149]: I'm not sure what the purpose of "pindoc :" is here. Would you please clarify what these occurrences are conveying?

File: ansys/dpf/core/operators/averaging/elemental_difference.py (hunk 3 above)
Comment (akaszynski) [reply ansys/dpf/core/operators/averaging/elemental_difference.py:35]: Returns for classes should be removed.  If these are attributes, we should be using an [Attributes](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring) Section.

File: ansys/dpf/core/operators/averaging/elemental_difference.py [hunk 8]
```diff
@@ -7,43 +7,58 @@
...
+    >>> op.inputs.mesh_scoping.connect(my_mesh_scoping)
+    >>> my_mesh = dpf.MeshedRegion()
+    >>> op.inputs.mesh.connect(my_mesh)
+    >>> my_through_layers = bool()
+    >>> op.inputs.through_layers.connect(my_through_layers)
+
+    >>> # Instantiate operator and connect inputs in one line
+    >>> op = dpf.operators.averaging.elemental_difference(field=my_field,mesh_scoping=my_mesh_scoping,mesh=my_mesh,through_layers=my_through_layers)
+
+    >>> # Get output data
```
Comment (akaszynski) [inline ansys/dpf/core/operators/averaging/elemental_difference.py:58]: Plain text should be separated and should not use `>>>` (which indicates code that should be executed).

```suggestion
    Get output data

```

Be sure this has a blank line above and below.

Comment (PipKat) [reply ansys/dpf/core/operators/averaging/elemental_difference.py:58]: @akaszynski Line 55 is similar. These are comments in their code. Should they not be?

**Comment by PipKat** [comment]
@akaszynski
How do we delete this PR now that we know that the PY files edited herein are not the source files?

**Comment by akaszynski** [comment]
> How do we delete this PR now that we know that the PY files edited herein are not the source files?

I've closed it, but you can do it too by clicking the "Close pull request" or "Close with comment" button.

**Pull request closed without merging**